toolchains/esp32/sdkconfig.defaults

```diff
@@ -74,19 +65,13 @@ CONFIG_MBEDTLS_SSL_IN_CONTENT_LEN=7800
 CONFIG_MBEDTLS_SSL_OUT_CONTENT_LEN=3700
 # CONFIG_MBEDTLS_HARDWARE_MPI is not set
 CONFIG_MBEDTLS_TLS_CLIENT_ONLY=y
-# CONFIG_MBEDTLS_SSL_PROTO_TLS1 is not set
-# CONFIG_MBEDTLS_SSL_PROTO_TLS1_1 is not set
-CONFIG_MBEDTLS_SSL_PROTO_DTLS=n
```

Comment:
these don't seem to exist on the right side anymore.
something to worry about?

Reply:
It looks like MbedTLS doesn't have support for older things that TLS 1.2 any more.